src/components/Contacts/ContactDetails/ContactDetailsTab/Tags/ContactTags.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Chip, Link, styled, Typography } from '@material-ui/core';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+const ContactTagsContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  alignContent: 'center',
+  flexWrap: 'wrap',
+  margin: theme.spacing(2, 0),
+  paddingLeft: '0',
+}));
+
+const ContactTagChip = styled(Chip)(({ theme }) => ({
+  margin: theme.spacing(1),
+  marginLeft: '0',
+}));
+
+const ContactTagAddTagText = styled(Typography)(({ theme }) => ({
+  margin: theme.spacing(1),
+  marginLeft: '0',
+}));
+
+interface ContactTagsProps {
+  contactId: string;
+  contactTags: string[];
+}
+
+export const ContactTags: React.FC<ContactTagsProps> = ({
+  contactId,
+  contactTags,
+}) => {
+  const { t } = useTranslation();
+  const handleTagDelete = (tag: string) => {
+    const index = contactTags.indexOf(tag);
+    if (index > -1) {
+      contactTags = contactTags.splice(index, 1);
+      // TODO: add mutation to update tag for contact
+    }
+    console.info('You clicked the Delete tag for tag' + tag + '' + contactId);
+  };
+  const handleAddTag = () => {
+    // TODO: add mutation to update tag for contact
+    console.info('You clicked add contact Tag to contact' + contactId);
+  };
+
+  return (
+    <ContactTagsContainer>
+      {contactTags.map((tag, _index) => (
```

Comment:
You don't need the ```_index``` if you're not using it. The only question would be, is there a chance there could be two tags with the same value? If so maybe using the ```index``` would be better.

Reply:
@TheNoodleMoose logically the tag should not be repeated that wouldn't make since.  I think if we handle duplication on the creation of new tags that should be enough.  Also this is logic that we are going to have to comeback to since we don't have the mutation call for contact set up yet.

Reply:
I went to mpdx.org and tested trying to add two of the same tag. It allows you to submit it but it doesn't duplicate the tag so we should be good.